test/functional/test_framework/wallet.py

```diff
@@ -65,15 +65,20 @@ def send_self_transfer(self, *, fee_rate=Decimal("0.003"), from_node, utxo_to_sp
         assert send_value > 0
 
         tx = CTransaction()
-        tx.vin = [CTxIn(COutPoint(int(utxo_to_spend['txid'], 16), utxo_to_spend['vout']))]
+        tx.vin = [CTxIn(COutPoint(int(utxo_to_spend['txid'], 16), utxo_to_spend['vout']), nSequence = seq_num)]
         tx.vout = [CTxOut(int(send_value * COIN), self._scriptPubKey)]
+        tx.nVersion = 2
```

Comment:
You're accepting a `nVersion` param but not using it?

Reply:
oops my bad 😅